src/renderer/components/database-list-item-metadata.jsx

```diff
@@ -62,24 +63,51 @@ export default class DbMetadataList extends Component {
     }
 
     return items.map(item => {
-      const isClickable = !!onSelectItem;
+      const isClickable = !!onDoubleClickItem;
+      const hasChildElements = !!onSelectItem;
       const title = isClickable ? 'Click twice to select default query' : '';
       const onDoubleClick = isClickable
+        ? onDoubleClickItem.bind(this, database, item)
+        : () => {};
+      const onSingleClick = hasChildElements
         ? onSelectItem.bind(this, database, item)
         : () => {};
 
       const cssStyle = {...STYLE.item};
       if (this.state.collapsed) {
         cssStyle.display = 'none';
       }
+
       return (
         <span
           key={item.name}
           title={title}
```

Comment:
Running the app I realized the title "Click twice to select..." is showing even when I am with the mouse over the columns name. Should be good change the title now to appear only over the table name.